### ISSUE TITLE
Incorrect note spacing when using harm (MEI) on staff 2 

### ISSUE THREAD

**Issue by peterkorgaard** [rism-digital/verovio#2722]
I'm in process of updating our use of Verovio from version 2.7.2-9184ded to version 3.9.0. In this process I discovered that some of our MEI files renders incorrectly when adding harm to staff 2. Here is an example:

![image](https://user-images.githubusercontent.com/22868391/159164244-41edc80b-b6e1-44dd-9ced-75e6e95cf8a6.png)

If I remove harm from staff 2 the problem disappears but I need the harm on top of of staff 2

![image](https://user-images.githubusercontent.com/22868391/159164415-d00e5881-fad6-427c-97cd-00847d22131f.png)

Here is my MEI data

```
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://www.music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<mei meiversion="3.0.0" xmlns="http://www.music-encoding.org/ns/mei">
    <meiHead></meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef key.sig="0" meter.count="4" meter.unit="4" xml:id="scoredef-0000000854245304">
                        <staffGrp xml:id="staffgrp-0000002135155953">
                            <staffDef clef.line="2" clef.shape="G" lines="5" n="1" xml:id="staffdef-0000000390779071"/>
                            <staffDef clef.line="4" clef.shape="F" key.mode="major"  lines="5" n="2" />
                        </staffGrp>
                    </scoreDef>
                    <section>
                        <section xml:id="section-0000000501596869">
                            <measure right="single" xml:id="measure-0000001162520689">
                                <staff n="1" xml:id="staff-0000001504550402">
                                    <layer n="1" xml:id="layer-0000001357129027">
                                        <rest dur="8"/>
                                        <note dur="8" oct="4" pname="c" xml:id="note-0000001362615758"/>
                                        <beam xml:id="beam-0000000689621276">
                                            <note dur="8" oct="4" pname="d" xml:id="note-0000000436546893"/>
                                            <note dur="8" oct="4" pname="e" xml:id="note-0000001827876088"/>
                                        </beam>
                                        <beam xml:id="beam-0000002138446682">
                                            <note dur="8" oct="4" pname="f" xml:id="note-0000000980998379"/>
                                            <note dur="8" oct="4" pname="g" xml:id="note-0000000819365625"/>
                                        </beam>
                                        <beam xml:id="beam-0000000218313685">
                                            <note dur="8" oct="4" pname="a" xml:id="note-0000000933616119"/>
                                            <note dur="8" oct="4" pname="b" xml:id="1EE0D40A-29D6-4DD5-AC9A-CCE564E4A4CB" ></note>
                                        </beam>
                                    </layer>
                                </staff>
                                <staff n="2">
                                    <layer n="1">
                                        <mRest xml:id="mrest-0000001896597482" />
                                    </layer>    
                                </staff> 
                            </measure>
                            <measure right="dbl" xml:id="measure-0000001658011779">
                                <staff n="1" xml:id="staff-0000001430331689">
                                    <layer n="1" xml:id="layer-0000000144620358">
                                        <note dur="4" oct="5" pname="c" xml:id="59B4AA08-7D91-4C06-B0BE-163D0918BECA" ></note>

                                        <chord dur="4" xml:id="chord-1">
                                            <note oct="5" pname="c" xml:id="note-0000001580004656">
                                               
                                            </note>
                                            <note oct="4" pname="g" xml:id="note-0000001580004656a"/>
                                            <note oct="4" pname="e" xml:id="note-0000001580004656s"/>
                                        </chord>
                                        <chord dur="4" xml:id="chord-1a">
                                            <note oct="5" pname="c" xml:id="note-0000001580004656qwe">
                                                
                                            </note>
                                            <note oct="4" pname="a" xml:id="note-0000001580004656aq1"/>
                                            <note oct="4" pname="f" xml:id="note-0000001580004656sw2"/>
                                        </chord>
                                        <chord dur="4" xml:id="chord-2">
                                            <note oct="4" pname="b" xml:id="note-0000001101551140">
                                                
                                            </note>

                                            <note oct="4" pname="g" xml:id="note-0000001101551140a"/>
                                            <note oct="4" pname="d" xml:id="note-0000001101551140s"/>
                                        </chord>
                                    </layer>
                                </staff>
                                <staff n="2">
                                    <layer n="1">
                                        <rest dur="4" />
                                       <note dur="4" oct="3" pname="c" xml:id="note-0000001580004656a"/>
                                       <note dur="4" oct="3" pname="f" xml:id="note-0000001580004656b1"/>
                                       <note dur="4" oct="3" pname="g" xml:id="note-0000001580004656b"/> 
                                    </layer>    
                                </staff> 
                                
                                <harm place="above" staff="1" tstamp="2">
                                    <rend fontsize="small">C</rend>
                                </harm>
                                <harm place="above" staff="1" tstamp="3">
                                    <rend fontsize="small">F</rend>
                                </harm>
                                 <harm place="above" staff="1" tstamp="4">
                                    <rend fontsize="small">G</rend>
                                </harm>
                                <harm place="above" staff="2" tstamp="2">
                                    <rend fontsize="small">1</rend>
                                </harm> 
                                <harm place="above" staff="2" tstamp="3">
                                    <rend fontsize="small">4</rend>
                                </harm>
                                <harm place="above" staff="2" tstamp="4">
                                    <rend fontsize="small">5</rend>
                                </harm>
                            </measure>
                        </section>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by rettinghaus** [comment]
Connected to #2675

**Comment by lpugin** [comment]
Fixed with 0dc6e7f5c6c0692a2b4b1b0cfbf66fdb3373f94d
